synapse/lib/snap.py

```diff
@@ -725,6 +725,11 @@ async def applyNodeEdits(self, edits):
                     node.bylayer['tags'].pop((tag, prop), None)
                     continue
 
+                if etyp == s_layer.EDIT_EDGE_ADD:
```

Comment:
Rather than paying this cost on edge adds (perf impact), i think we can probably populate the hive when the docs are set for the first time